performant-software/Neatline
# Editor map layering and presenter fixes

### What does this PR do?
- Addresses issues with the appearance/usability of the drawing tool icon and other map objects
- Avoids an error that happened when switching to the "Map" tab of a selected item

### What issues does it address?
- Addresses #393, addresses #394 

### How to test?
- Login to http://neatline-staging.herokuapp.com/admin/, click Neatline and open one of the maps
- Click on an existing record link or its map object to select it. In the editor pane to the left, click on the Map tab. Then switch the tool to "Draw Point" by selecting it under Drawing Tools
- Move the cursor around the map section of the page. An icon should follow it to indicate what a point would look like if added. This icon should always appear above other objects, and its opacity should be greater than that of existing points for the record (if the record has opacity values set to less than 1.0 in its style section).
- Try this for various records and when toggling between different drawing tool selections. The Draw Point, Draw Line, and Draw Polygon tools should all display the cursor-follower this same way.
